elide-core/src/main/java/com/yahoo/elide/core/EntityDictionary.java

```diff
@@ -669,7 +669,10 @@ public String getNameFromAlias(Object entity, String alias) {
      * @return true if entityClass is shareable.  False otherwise.
      */
     public boolean isShareable(Class<?> entityClass) {
-        return getAnnotation(entityClass, SharePermission.class) != null;
+        return entityHasChecksForPermission(
```

Comment:
Does the first part of this statement make sense anymore or must everything be explicitly _marked_ shareable now?

Reply:
So basically entity with `@SharePermission` is shareable, **No SharePermission**=**Not sharable**, also `@SharePermission(false)`=not shareable

Reply:
How about something like:
```
Annotation shareAnnotation = getAnnotation(entityClass, SharePermission.class)
return (shareAnnotation !=null && shareAnnotation.shareable())
```